### PR TITLE
fix MPT inference inaccuracy output

### PR DESCRIPTION
**Pull request by astachowiczhabana**
call nn.functional.softmax directory instead of class Softmax()

### CODE REVIEW

**Comment by regisss** [comment]
I think this is a duplicate of #1857

**Comment by astachowiczhabana** [comment]
yes, closing
